src/components/Items/Item.tsx

```diff
@@ -32,7 +32,7 @@ const Item: React.FC<ItemProps> = ({ index, item }) => {
       ? setTimeout(() => {
           if (!manifest)
             vault
-              .loadManifest(item.id)
+              .load(item.id)
```

Comment:
Switches vault to load IIIF resource as either a **ManifestNormalized** or **CollectionNormalized**